examples/40_advanced/example_resampling.py

```diff
@@ -0,0 +1,114 @@
+"""
+=======
+Holdout
+=======
+
+In *auto-sklearn* it is possible to use different resampling strategies
+by specifying the arguments ``resampling_strategy`` and
+``resampling_strategy_arguments``. The following example shows common
+settings ``AutoSklearnClassifier``.
```

Comment:
shouldn't;t it be common setting **about**? Or do you mean the recommended setting to use on this estimator?

Reply:
Indeed, that's wrong.